docs/pages/blog/turbo-1-4-0.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Turborepo 1.4
+date: 2022/07/29
+description: Turborepo 1.4 has new examples, automatically includes environment variables, and more!
+tag: web development
+ogImage: /images/blog/turbo-1-4-0/og-image.png
+---
+
+# Turborepo 1.4
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Tuesday, August 9th, 2022</div>
+
+<Authors authors={[ 'gsoltis', 'nathanhammond', 'tknickman', 'anthonyshewdev' ]} />
+
+Turborepo 1.4 improves usage with frameworks and adds new examples:
+
+- [**Automatic environment variable inclusion:**](#automatic-environment-variable-inclusion): We'll automatically infer popular framework environment variables for you. No need to declare them yourself in `turbo.json`.
+- [**`eslint-plugin-turbo`**](#eslint-plugin-turbo): Enhanced feedback with a new ESLint plugin
+- [**New framework and library examples:**](#new-framework-and-library-examples) New starters and examples requested by the community.
+
+Update today by running `npm install turbo@latest`.
+
+## Automatic environment variable inclusion
+
+To help ensure correct caching across environments, Turborepo will now automatically infer and include public environment variables when calculating cache keys for apps built with Astro, Create React App, Gatsby, Next.js, Nuxt, SvelteKit, Vite, Vue, and more. You can safely remove these framework-specific public environment variables from `turbo.json` if you have manually declared them.
+
+```diff filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+        "^build"
+-       // Include build time public inlined environment variables that
+-       // are different in development and production, so that
+-       // Turborepo does not use the same cached build
+-       // accross environments
+-       "$NEXT_PUBLIC_EXAMPLE_ENV_VAR"
+      ]
+    }
+  }
+}
+```
+
+Note that this automatic detection and inclusion only works if Turborepo successfully infers the framework your apps are built with. Additionally, environment variables will only be part of the cache key in the workspaces where that framework is used. In other words, environment variables inferred for Next.js apps, will only apply to the cache key for workspaces inferred as Next.js apps. Other workspaces in the monorepo will not be impacted.
+
+For example, consider a monorepo with a three workspaces: a Next.js project, a Create React App project, both depending on a shared Typescript package and each with their own`build` script. Let's say that this Turborepo has a standard `turbo.json` pipeline that builds them all in order:
+
+```jsonc filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+				"^build",
+			]
+    }
+}
+```
```

Comment:
source looks ok, but indent is weird. not a blocker
<img width="339" alt="Screen Shot 2022-08-08 at 4 51 25 PM" src="https://user-images.githubusercontent.com/490968/183533683-0f7e4a0b-ba25-45d2-9c81-08ffc5a56b97.png">